text/1055-vanilla-prettier-setup-in-blueprints.md

```diff
@@ -0,0 +1,93 @@
+---
+stage: accepted
+start-date: 2024-12-03T00:00:00.000Z
+release-date: 
+release-versions:
+teams:
+  - cli
+prs:
+  accepted: https://github.com/emberjs/rfcs/pull/1055
+project-link:
+suite: 
+---
+
+# Vanilla Prettier Setup in Blueprints
+
+## Summary
+
+This RFC proposes to migrate to a vanilla Prettier setup in the blueprints, instead of running Prettier via ESLint and Stylelint.
+
+This RFC also proposes to not include any custom values for specific Prettier options, and instead truly rely on Prettier's defaults for formatting.
```

Comment:
What's the reasoning for coupling both? I'm highly in favor of a vanilla Prettier setup. Upgrading Prettier is often ending in a dependency hell due to current setup. But I'm uncertain about the change to double quotes due to churn in the ecosystem. Maybe we could decouple both and discuss them in separation?

Reply:
No particular reason, but it seemed to make sense to me to also review the current Prettier config. To me, it doesn't make much sense for new projects to start of with a mix of single and double quotes across different files. It'd be nice if new projects embrace Prettier's philosophy, and just use the defaults. Existing projects can ignore the config update, or run `lint:format:fix` to fix all formatting issues in one go.
Does't have to be a separate RFC for me, I'm also fine with not doing this, as mentioned in `Alternatives`.

Reply:
I must admit that it may be only my personal preference due to being used to a specific style which makes me struggle with that change. In theory I fully agree that Ember should not try to overrule established defaults in the ecosystem.

Reply:
I realize that part of this RFC might be a bit "radical" though, so I'm completely fine with dropping it.

Reply:
I would personally be in favour of dropping this from the RFC and splitting it out to another RFC if you feel strongly about it. 

I feel like we need to have a larger conversation about things like `Existing projects can ignore the config update` and blueprint updates, and this RFC is a slam dunk without that one "radical" change 😂

Reply:
Also in favor of splitting this proposal into a separate RFC.

In theory, I agree that Ember shouldn't diverge from ecosystem defaults here (though the first thing I'd do is add back in the current config bc it's how my brain is trained).

In that separate PR, I'm going to push back on:

> Existing projects can ignore the config update, or run `lint:format:fix` to fix all formatting issues in one go.

But we can save that discussion for later.